credsweeper/scanner/scan_type/scan_type.py

```diff
@@ -53,8 +53,6 @@ def filtering(cls, config: Config, target: AnalysisTarget, line_data: LineData,
             If `use_filters` option is false, always return False
 
         """
-        if not config.use_filters:
-            return False
```

Comment:
Is it removed due to supporting `MultiPattern`?

Reply:
No. the check is moved to  ``get_line_data``.
When first candidate is filtered - second might be not filterd.

Reply:
IMO `MultiPattern` should be `and` condition, not `or`.
`MultiPattern` is meaningful when both patterns are detected..

Reply:
Yes, ``MultiPattern`` requires two patterns (from values) match https://github.com/babenek/CredSweeper/blob/c9926c8ff69dd304033b0a2d8e2633db02cec6ce/credsweeper/scanner/scan_type/scan_type.py#L111
Workaround in the PR is to fix case when two different credentials are in single line. Currently Credsweeper cannot find second with one rule.

Reply:
Let me show you: https://github.com/babenek/CredSweeper/commit/2a56e8117347f83c2eda39863008446c4e600c45 - dummy key hides real key.
Suppose, there might be a bit bigger refactoring ...

Reply:
Ah.. I see.

So, If there are more than 2 detected patterns in one line, and if the first detected pattern just filtered then the others skipped also.
That's why you move the `if` condition into the loop, I understood.